README.md

```diff
@@ -0,0 +1,54 @@
+# RHVoice
+
+RHVoice is a free and open source speech synthesizer.
+
+## Features
+
+### Speech synthesis method
+
+RHVoice uses statistical parametric synthesis. It relies on existing
+open source speech technologies (mainly
+[HTS](http://hts.sp.nitech.ac.jp) and related software).
+
+Voices are built from recordings of natural speech. They have small
+footprints, because only statistical models are stored on users'
+computers. And though the voices lack the naturalness of the
+synthesizers which generate speech by combining segments of the
+recordings themselves, they are still very intelligible and resemble
+the speakers who recorded the source material.
+
+### Supported languages
+
+Initially, RHVoice could speak only Russian. Now it also supports:
+
+* American English
+* Brazilian Portuguese
+* Esperanto
+* Georgian
+* Ukrainian
+* Kyrgyz
+* Tatar
+
+In theory, it is possible to implement support for
+other languages, if all the necessary resources can be found or
+created.
+
+### Supported platforms
+
+RHVoice supports folowing platforms:
+
+* Windows
+* GNU/Linux
+* Android
+
+It is compatible with
+standard text-to-speech interfaces on these platforms: SAPI5 on
+Windows, [Speech Dispatcher](http://devel.freebsoft.org/speechd) on
+GNU/Linux and Android's text-to-speech APIs. It can also be used by
+the [NVDA screen reader](http://www.nvaccess.org) directly (the driver
+is provided by RHVoice itself).
```

Comment:
I would like to read more about Linux text-to-speech API from here, because I don't know.

Reply:
what about

> [Speech Dispatcher](http://devel.freebsoft.org/speechd)

line 46

Reply:
I misread "on GNU/Linux" as "or GNU/Linux". Need to be more attentive with fast reviews. =)